src/KeyVault/KeyVault/Commands/Certificate/ImportAzureKeyVaultCertificate.cs

```diff
@@ -88,6 +88,15 @@ public class ImportAzureKeyVaultCertificate : KeyVaultCmdletBase
                    HelpMessage = "Base64 encoded representation of the certificate object to import. This certificate needs to contain the private key.")]
         public string CertificateString { get; set; }
 
+        /// <summary>
+        /// Specifies type of the certificate to be imported.
+        /// </summary>
+        [Parameter(Mandatory = false,
+                   ParameterSetName = ImportWithPrivateKeyFromStringParameterSet,
+                   HelpMessage = "Specifies the type of the certificate to be imported. Regards certificate string as PFX format by default.")]
+        [PSArgumentCompleter(Constants.Pkcs12ContentType, Constants.PemContentType)]
+        public string ContentType { get; set; } = Constants.Pkcs12ContentType;
```

Comment:
I'm thinking whether we should expose the raw content type like `-ContentType application/x-pkcs12` or an enum (or string) to represent the content type for example `-ContentType Pkcs12` and translate the enum to the actual content type before calling SDK. The latter syntax is definitely simpler to use.
What does CLI's interface look like?

Reply:
The reason why I didn't do this translation is I don't want to maintain a mapping list for this kind of content type.

CLI hasn't supported importing certificate from string directly. So they didn't expose this parameter. 

To align with Portal, I agree we can use wrapped value to represent the content type. 

![image](https://user-images.githubusercontent.com/19869716/183621818-14ed30d2-36d5-4d1a-8c22-32cdde7507e5.png)

Reply:
I vote for "PKCS12" and "PEM" because they are much clearer to use.

Reply:
Has exposed these raw values to customer in New/Set-AzKeyVaultCertificatePolicy. Let us keep align with previous design.